libs/common/domain/src/lib/model/record/metadata.model.ts

```diff
@@ -159,7 +159,8 @@ export interface GraphicOverview {
 }
 
 export interface DatasetSpatialExtent {
-  geometry: Geometry
+  geometries?: Geometry[]
+  bbox?: [number, number, number, number]
```

Comment:
So now I understand what you meant by using a `bbox` property.

My original approach with this field was to describe simple bboxes as GeoJSON geometries as well (simple polygons). See for example the mapper for the GN4 index format:

https://github.com/geonetwork/geonetwork-ui/blob/785657302d93b46785c0eb143a60ef4186934208/libs/api/metadata-converter/src/lib/gn4/gn4.field.mapper.ts#L292-L316

What do you think about getting rid of the `bbox` property and only use the `geometries` one?